docs/.vitepress/theme/index.ts

```diff
@@ -1,28 +1,24 @@
-import { h, onMounted, watch, nextTick } from 'vue'
+import { onMounted, watch, nextTick } from 'vue'
```

Comment:
在 `docs/.vitepress/theme/index.ts` 中，您重新组织了导入语句并更新了布局组件。建议确认所有组件都已正确注册并且没有遗漏。

```suggestion
import { onMounted, watch, nextTick } from 'vue'
import type { Theme } from 'vitepress'
import { useRoute, inBrowser } from 'vitepress'
import DefaultTheme from 'vitepress/theme'
import './style/index.css'
import '@fortawesome/fontawesome-free/css/all.min.css'
import mediumZoom from 'medium-zoom'
import busuanzi from 'busuanzi.pure.js'
import BilibiliVideo from './components/BilibiliVideo.vue'
import bsz from "./components/bsz.vue"
import Linkcard from "./components/Linkcard.vue"
import ArticleMetadata from "./components/ArticleMetadata.vue"
import HomeUnderline from "./components/HomeUnderline.vue"
import DownloadPage from './components/DownloadPage.vue'
import MyLayout from './components/MyLayout.vue'